aspnetcore/security/authentication/social/microsoft-logins.md

```diff
@@ -4,25 +4,27 @@ author: rick-anderson
 description: This sample demonstrates the integration of Microsoft account user authentication into an existing ASP.NET Core app.
 ms.author: riande
 ms.custom: mvc
-ms.date: 05/11/2019
+ms.date: 12/03/2019
```

Comment:
ms.date: 12/5/2019

Reply:
@scottaddie  try suggesting any line. I did the following only to demo the problem